src/llm_tracekit/litellm/instrumentor.py

```diff
@@ -35,44 +35,23 @@ def __init__(
         application_name: Optional[str] = None,
         subsystem_name: Optional[str] = None,
     ):
-        self._config: Optional[OpenTelemetryConfig] = self.generate_exporter_config(
+        self._config: Union[ExportConfig, LiteLLMConfig, None] = generate_exporter_config(
```

Comment:
since you're setting `self.config` after this, do `config = generate_exporter_config(...)`.
Also, the type hint shouldn't be necessary